File: CG/training/easy/la descente.js
var arr = []

while (true) {
    for (var i = 0; i < 8; i++) {
        var mountainH = parseInt(readline()); // represents the height of one mountain, from 9 to 0.
        arr.push(mountainH)
    }
    
    posi = arr.indexOf(Math.max.apply(Math, arr))
    arr = []
    print(posi);

    // To debug: printErr('Debug messages...');
}

File: CG/training/easy/la descente.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    while (1) {
        int index = 0;
        int max = 0;
        for (int i = 0; i < 8; i++) {
            int mountainH; // represents the height of one mountain.
            cin >> mountainH; cin.ignore();
            if (mountainH > max) {
                max = mountainH;
                index = i;
            }
        }

        // Write an action using cout. DON'T FORGET THE "<< endl"
        // To debug: cerr << "Debug messages..." << endl;

        cout << index << endl; // The index of the mountain to fire on.
    }
}